source/iterators.tex

```diff
@@ -1346,8 +1346,8 @@
 \libconcept{indirectly_writable} has the awkward \keyword{const_cast} expressions to reject
 iterators with prvalue non-proxy reference types that permit rvalue
 assignment but do not also permit \keyword{const} rvalue assignment.
-Consequently, an iterator type \tcode{I} that returns \tcode{std::string}
-by value does not model \tcode{\libconcept{indirectly_writable}<I, std::string>}.
+Consequently, an iterator type \tcode{I} that returns \tcode{string}
+by value does not model \tcode{\libconcept{indirectly_writable}<I, string>}.
```

Comment:
The purpose of the note is to clarify the normative text. Arguably `string` in this context is less clear than `std::string`. A clarifying note should be as clear as possible, and we can be more lax about formatting/specification rules in notes. I don't feel strongly about it though.

The other part of this fix looks fine

Reply:
Reasonable, thanks for the suggestion.